basic-ellipsis/public/index.html

```diff
@@ -1,65 +1,125 @@
-    </div>
-    <p id="load">Firebase SDK Loading&hellip;</p>
-
-    <script>
-      document.addEventListener('DOMContentLoaded', function() {
-        // // 🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥
-        // // The Firebase SDK is initialized and available here!
-        //
-        // firebase.auth().onAuthStateChanged(user => { });
-        // firebase.database().ref('/path/to/ref').on('value', snapshot => { });
-        // firebase.messaging().requestPermission().then(() => { });
-        // firebase.storage().ref('/path/to/ref').getDownloadURL().then(() => { });
-        //
-        // // 🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥
-
-        try {
-          let app = firebase.app();
-          let features = ['auth', 'database', 'messaging', 'storage'].filter(feature => typeof app[feature] === 'function');
-          document.getElementById('load').innerHTML = `Firebase SDK loaded with ${features.join(', ')}`;
-        } catch (e) {
-          console.error(e);
-          document.getElementById('load').innerHTML = 'Error loading the Firebase SDK, check the console.';
-        }
-      });
-    </script>
+        <header class="wrapper">
+          <div class="content_container content_container__without_border content_container__text_secondary_color">
+            <img src='images/Label.png' alt='label' class="header__logo" />
+            <h1 class="header__header">WORKING<br aria-hidden="true">WITH design LAB </h1>
+            <p class="header__text">
+              We’ve been helping startups launch and grow their businesses.
+              We help you to fill the gap between development, 
+                design and product management.
+            </p>
+            <img src='images/Mouse-icon.svg' alt='mouse image' class="header__mouse-logo" />
+          </div>
+        </header>
+        <div class="wrapper">
+          <div class="content_container">
+            <h2 class="content_container__header">About us</h2>
+            <p class="content_container__text">
+              When you work with us, you are part of a team of committed collaborators.
+              All aspects of our software development process are integrated, iterative and agile.
+               
+              Our approach unifies design, development and
+              product management to create exceptional products.
+              We start each project by understanding your business goals,
+              the user’s needs and the product requirements.
+              Then we help translate them into effective and elegant solutions.
+            </p>
+          </div>
+        </div>
+        <div class="wrapper wrapper__theme_services">
+          <div class="content_container content_container__services content_container__text_secondary_color content_container__border_secondary_color">
+            <h2 class="content_container__header content_container__services__header content_container__header__without_background"><span>Services</span></h2>
```

Comment:
there shouldn't be element of elements, means __element__element
__ = for element
_ = for modifier